soundfile.py

```diff
@@ -677,11 +819,13 @@ def __init__(self, file, mode='r', samplerate=None, channels=None,
         """
         # resolve PathLike objects (see PEP519 for details):
         # can be replaced with _os.fspath(file) for Python >= 3.6
-        file = file.__fspath__() if hasattr(file, '__fspath__') else file
+        if isinstance(file, _os.PathLike):
```

Comment:
That's raising the compatibility to >= 3.6, right?

Reply:
Just forgot to remove this comment.
https://github.com/bastibe/python-soundfile/pull/462/commits/0f4379c16898f428f689f8c142365f747f90602d